lib/rules/require-data-selectors.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview Don't use classes as an identifier
+ * @author Sandeep Baldawa
+ */
+'use strict'
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Use data-* attributes to provide context to your selectors and insulate them from CSS or JS changes https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements',
+      category: 'Possible Errors',
+      recommended: false,
+      url: 'https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements',
+    },
+    schema: [],
+    messages: {
+      unexpected: 'Do not use dynamic ids or classes for elements',
```

Comment:
should be "use data-* attribute selectors instead of classes or tag names"

Reply:
addressed the comment, thanks!